### ISSUE TITLE
Auto run animation when activity start

### ISSUE THREAD

**Issue by tronghieubk** [ZieIony/Carbon#25]
I have a Button like this, at the start of activity I want to have all run popup animation, but I cannot find anyway to achieve this.

< carbon.widget.Button
                    android:id="@+id/new_task_enter_number"
                    style="@style/contact_select_btn"
                    android:text="{fa-plus} Enter" 
                    app:carbon_inAnimation="pop"
                    app:carbon_outAnimation="pop"/>

I have tried set view to Button "gone" (in both code and xml) and change to "visible" in code but no luck. Thanks in advance.

**Comment by ZieIony** [comment]
Could you please elaborate? The button is visible, but not animating, right? Did you try to launch the animation later in the app, for example in an OnClickListener? What device and Android version are you using?

My guess is that the Activity creation process is taking too long and the animation goes unnoticed. You can try launching the animation in onResume or even later.

i will try to reproduce your case as soon as possible.

**Comment by tronghieubk** [comment]
Hi Zielony,

My goal is to start an blank activity and then all the vies appear. 

I have tried to change the visibility in onResume but the first time the view state changes from "gone" to "visible" it won't be animated. Even if the view state is controlled by an onClickListener, the first time that view goes visible there is no animations but after that when I continue clicking on the button, the view pop in and out perfectly.

**Comment by ZieIony** [comment]
I was able to reproduce your issue. The appearing animation changes alpha value from current to 1. Invisible views have alpha value set to 1, because visibility is a flag, not an alpha value. The workaround for this one is to set alpha to 0. You can do that in xml for more recent phones or using nineolddroids for older devices.

```
android:alpha="0"

button.setAlpha(0.0f);

ViewHelper.setAlpha(button,0);
```

I'm marking this issue as a bug. Thanks for finding this!

**Comment by ZieIony** [comment]
fixed in 0.6.5
